components/dashboard/ExpenseReporter/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { gql, useQuery } from '@apollo/client'
+import React from 'react'
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+
+import DateHelper from '../../../lib/date-helper'
+
+function ExpenseReporter() {
+  const today = new Date()
+  const lastSixMonth = DateHelper.dateBeforeSixMonths()
+
+  const urlLink = `?startmonth=${today.getFullYear()}${today.getMonth()}&endmonth=${lastSixMonth.getFullYear()}${lastSixMonth.getMonth()}`
+
+  const mutateQuery = (urlParams: string) => gql`
+    query MajorPlannedExpenses {
+      expenses @rest(type: "MajorPlannedExpenses", path: "${urlParams}", endpoint: "majorPlannedExpensesWorkItemwise")
```

Comment:
Make a GQL call to the new Apollo server now instead of making REST API call from the frontend